### PR TITLE
DBZ-2122 Add Db2 and SQL Server topics about capture job configuration

### PR DESCRIPTION
**Pull request by roldanbob**
Jira [DBZ-2122](https://issues.redhat.com/browse/DBZ-2122)

Adds capture job configuration for Db2. Although the information about the parameters is specific to Db2, the bulk of this material is conceptual and should be ready for reuse more or less as-is in sqlserver.adoc. I'll add a version of this material to the SQL Server connector doc after we agree on the conceptual content.

**UPDATE**:

Adds capture job configuration content for Db2 and SQL Server.

Tested in local upstream Antora build and downstream asciidoctor build.

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/db2.adoc [hunk 1]
```diff
@@ -1427,6 +1427,44 @@ VALUES ASNCDC.ASNCDCSERVICES('reinit','asncdc');
...
 {link-prefix}:{link-db2-connector}#managing-debezium-db2-connectors[Reference table for {prodname} Db2 management UDFs]
 
+// Type:concept
+// ModuleID: how-the-db2-capture-agent-configuration
+=== How the Db2 capture agent configuration affects latency, server load, and performance
+
+When a database administrator enables change data capture for a source table, the capture agent begins to run.
+The agent reads new change event records from the transaction log and replicates the event records to a capture table.
+Between the time that a change is committed in the source table, and the time that the change appears in the corresponding change table, there is always a small latency interval.
+This latency interval represents a gap between when changes occur in the source table and when they become available for {prodname} to stream to Kafka.
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/db2.adoc:1437]:
```suggestion
This latency interval represents a gap between when changes occur in the source table and when they become available for {prodname} to stream to Apache Kafka.
```

**Review by gunnarmorling — APPROVED** [review]
@jpechane, any comments on this one?

**Comment by roldanbob** [comment]
Added the SQL Server version of the material. This ~is~ should now be complete. Let me know if any further updates are required. Thanks!

**Comment by gunnarmorling** [comment]
Thanks a lot, @roldanbob!

**Pull request merged**